docs/DataFeeds.md

```diff
@@ -1,5 +1,8 @@
 # Data Feeds
 ## KLASS
+Data is imported as JSON from https://data.ssb.no/api/klass/v1, (documented here: https://data.ssb.no/api/klass/v1/api-guide.html).
+For municipal facts we use KLASS as a source for counties and municipalities. 
+The output for counties is typed up [here](/src/main/resources/lib/klass/counties.ts) and municipalities [here](/src/main/resources/lib/klass/municipalities.ts)
```

Comment:
I would advise not using "here" links, it's better to use actual filename or anything else really.